src/TodoApp.js

```diff
@@ -0,0 +1,334 @@
+const addBubblingEvent = function(eventName, eventTarget, callback) {
+	document.body.addEventListener(eventName, function(e) {
+		if(e.target === eventTarget)
+			callback(e);
+	});
+};
+
+function TodoApp() {
+	const todoItems = {
+		active: [],
+		completed: []
+	};
+
+	const list = document.querySelector("#todo-list");
+	const countBox = document.querySelector(".todo-count");
```

Comment:
```suggestion
	const $countBox = document.querySelector(".todo-count");
```
이게 좋은건진 모르겠지만, 돔요소 선택한거는 로직을 위한 상수랑 구분하기 위해서 변수 이름 앞에 $붙이기도 한다고 하더라구요! 참고하시면 좋을 것 같아요~!ㅎㅎㅎ

Reply:
충고 감사합니다!